src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -805,6 +805,12 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
                             {
                                 AddDiagnostic(diagnostics, ErrorCode.ERR_SwitchNeedsString, MessageID.IDS_Text.Localize(), "/langversion:");
                             }
+                            else if (value.StartsWith("0"))
```

Comment:
Please use an explicit comparer here.  The default for `StartsWith` is to use culture sensitive comparisons and I'm unsure if that is correct here.  Believe this should be ordinal.